lib/ntbb-session.lib.php

```diff
@@ -1,12 +1,12 @@
 <?php
 
 require_once dirname(__FILE__) . '/ntbb-database.lib.php';
+include_once '../../pokemonshowdown.com/config.inc.php';
```

Comment:
What does it need from main config? It should probably be moved into Play config.

Reply:
It needs `$psclient['routes']`. What's this "Play config" you mention?

Reply:
Do you mean `$psconfig['routes']`? That's defined in `play.pokemonshowdown.com/config.inc.php`, not `pokemonshowdown.com/config.inc.php`, right?

Reply:
Oh, right, it's there - the config paths were a bit hard to wrap my head around. I'll fix that.

Reply:
TBH, i think there should probably only be one config. Currently its kind of a mess (pokemonshowdown.com sometimes includes both in places and overwrites the $psconfig global which is ick)

Reply:
I agree. I think the two config files should be merged and moved into `play.pokemonshowdown.com/config/config.inc.php`.

Reply:
I think maybe we can punt on that and do it in a follow up? Cross repo stuff like this is awkward and might bog down this PR. We definitely all agree I think that it should be done

Reply:
Yeah, it doesn't need to be in this PR.

Reply:
In general if I suggest something unrelated in PR comments, I'm not actually saying it needs to be in this particular PR.